package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "port-github-action",
-	"version": "1.1.0",
+	"version": "1.1.1",
```

Comment:
Since we've bumped the version, we may need to verify if the development-team will need to adjust anything at their end in order to benefit from this bugfix

Reply:
Okay